<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>295</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string notr="true">Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-calculator"/>
  </property>
  <property name="windowOpacity">
   <double>0.850000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QPushButton" name="add4PushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="multiplyPushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="add3PushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="equalPushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="add2PushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="add8PushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="add1PushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="add7PushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="minusPushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="addDotPushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="add9PushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="add5PushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="plusPushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QTextEdit" name="outputLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="add6PushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="dividePushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="add0PushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="clearPushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="delPushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="openParenthesisPushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="closeParenthesisPushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(110, 110, 110);
  color: white; 
  border: 1px solid white;
  border-radius: 18px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
